README.md

```diff
@@ -115,6 +118,19 @@ e.g. by standardizing Kubernetes labels using an
 [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
 that ensures that there are no possible conflicts.
 
+#### Enabling Kubernetes labels
+
+In order to prevent high cardinality of the `*_labels` family of metrics, kube-state-metrics only exposes specifically allowed labels.
+The labels can be configured through the use of the `--labels-allow-list` flag. The flag accepts a comma separated
+list of `<metric>=[<labels>]`, where `<labels>` are again comma separated full label names. For example:
+
+```
+--labels-allow-list kube_deployment_labels[label_app,label_env,deployment,namespace],kube_pod_labels[label_app,label_env,pod,namespace]
+```
+
+Note that the current implementation requires the inclusion of all wanted labels, this includes the metrics own labels like `deployment`, `node`, `pod`, etc. and `namespace`.
```

Comment:
This can be removed as mentioned above.

Reply:
Yeah, I figured. I was wondering whether the fix for that bug would need more time, hence the "current implementation", as the bugfix could always patch the documentation. Regardless, I've updated the PR with the suggestions.